NavigationReactNative/src/TabBar.tsx

```diff
@@ -112,4 +117,4 @@ const styles = StyleSheet.create({
     },
 });
 
-export default TabBar;
+export default TabBar;
```

Comment:
Can you the newline back so this line doesn't show up in the diff, please?

Reply:
This _should_ be fixed now. I think it's something about the way my editor is handling the EoF. If I add lines, it seems to modify the final line as well. Very annoying.